templates/header.hbs

```diff
@@ -1,27 +1,32 @@
 <a class="skip-navigation" tabindex="1" href="#main-content">{{t 'skip_navigation' }}</a>
 
 <header class="header">
-  <div class="logo">
-    {{#link 'help_center'}}
-      <img src="{{settings.logo}}" alt="{{t 'home_page' name=help_center.name}}" />
-      {{#if settings.show_brand_name}}
-        <span aria-hidden="true">{{help_center.name}}</span>
-      {{/if}}
+  <div class="header-left">
+    <div class="logo">
+      <a href={{settings.xpro_link}} class="xpro-link"> 
+        <img src="{{settings.logo}}" class="site-logo" />
+      </a>
+    </div>
+    {{#link 'help_center' class="support-center"}}
+      SUPPORT CENTER
     {{/link}}
+
   </div>
 
   <div class="nav-wrapper-desktop">
     <nav class="user-nav" id="user-nav" aria-label="{{t 'user_navigation'}}">
       <ul class="user-nav-list">
-        <li>{{link 'community'}}</li>
-        <li>{{link 'new_request' class='submit-a-request'}}</li>
-        {{#unless signed_in}}
-          <li>
-            {{#link "sign_in" class="sign-in"}}
-              {{t 'sign_in'}}
-            {{/link}}
-          </li>
-        {{/unless}}
+{{!--         <li>{{link 'community'}}</li> --}}
+        <li>{{#link 'new_request' class='submit-a-request'}}Contact Us{{/link}}</li>
```

Comment:
This has an underline on hover, Whereas the live theme doesn't. https://xpro.zendesk.com/hc/en-us

Reply:
It seems there is an underline on live theme:

<img width="1280" alt="Screenshot 2024-06-13 at 3 06 06 PM" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/1b08d097-21d2-4763-8c5a-982593b609d3">

Reply:
Sorry, I was talking about the `Submit` button above the footer. 
<img width="602" alt="image" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/34372316/e455c8c1-c43c-428d-be75-334f8606047b">